assets/sass/front-page.scss

```diff
@@ -200,90 +200,25 @@ $monitor-height: 271px;
 }
 
 #companies-projects {
-  padding-top: 20px;
-  clear: both;
   border-top: solid 1px var(--base-border-color);
 
   ul {
     @extend .unstyled !optional;
 
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
+
     li {
-      display: block;
-      float: left;
-      width: 108px;
-      height: 65px;
-      margin: 0 4px 25px 5px;
     }
 
     a {
-      display: block;
-      width: 108px;
-      height: 65px;
-      text-indent: -9999px;
-      background-repeat: no-repeat;
-      background-position: 0 0;
       opacity: 0.85;
       transition-duration: 0.3s;
       transition-property: opacity;
 
       &:hover {
         opacity: 1;
       }
-
-      &.google {
-        @include background-image-2x($baseurl + "images/company-project-logos/google", 108px, 65px);
```

Comment:
Probably makes sense to delete those images, too.

Reply:
I did, didn't I?

Reply:
Yes. I missed this somehow, sorry!